scripts/exglobal_archive.py

```diff
@@ -28,10 +28,11 @@ def main():
             'ARCH_GAUSSIAN_FHMAX', 'ARCH_GAUSSIAN_FHINC', 'ARCH_GAUSSIAN_FHINC',
             'DOIAU', 'OCNRES', 'ICERES', 'NUM_SND_COLLECTIVES', 'FHOUT_WAV',
             'FHOUT_HF_WAV', 'FHMAX_WAV', 'FHMAX_HF_WAV', 'FHMAX_WAV_GFS',
-            'restart_interval_gdas', 'restart_interval_gfs',
+            'restart_interval_gdas', 'restart_interval_gfs', 'DO_EXTRACTVARS',
```

Comment:
I don't think `DO_EXTRACTVARS` is used.
```suggestion
            'restart_interval_gdas', 'restart_interval_gfs',
```